react/javascript/demo-embeds/src/hooks/use_all_looks.jsx

```diff
@@ -31,10 +31,11 @@ import { sortByTitle } from './utils'
 
 const all = async (coreSDK) => {
   try {
-    const data = await coreSDK.ok(coreSDK.all_looks())
+    const data = (await coreSDK.ok(coreSDK.all_looks('id,title'))) || []
     data.sort(sortByTitle)
     return data
-  } catch (err) {
+  } catch (error) {
+    console.error(error)
     throw new Error('Error retrieving looks')
```

Comment:
do you want to keep this `console.error()` call? I guess your throw is to prompt the user to look at the console output?